src/SOS/SOS.Extensions/DebuggerServices.cs

```diff
@@ -537,7 +540,7 @@ private readonly unsafe struct IDebuggerServicesVTable
             public readonly delegate* unmanaged[Stdcall]<IntPtr, out uint, int> GetCurrentThreadSystemId;
             public readonly delegate* unmanaged[Stdcall]<IntPtr, uint, int> SetCurrentThreadSystemId;
             public readonly delegate* unmanaged[Stdcall]<IntPtr, uint, out ulong, int> GetThreadTeb;
-            public readonly delegate* unmanaged[Stdcall]<IntPtr, uint, uint, byte*, int> VirtualUnwind;
+            public readonly delegate* unmanaged[Stdcall]<IntPtr, uint, int, byte*, int> VirtualUnwind;
```

Comment:
Does this signature need to match the private VirtualUnwind in CorDebugDataTargetWrapper.cs and DataTargetWrapper.cs and LLDBServices.cs, which still use uint?

Reply:
I made them all consistent (int contextSize).